app/webpacker/packs/school_search.js

```diff
@@ -0,0 +1,35 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const axios = require("axios");
```

Comment:
There should probably be a linting rule for this, but it's generally bad practice to mix `import` and `require` unless there's a specific reason you're doing it

Reply:
More than happy to incorporate any suggestions - not a javascript specialist so I will rely on any input

Reply:
I prefer using `import`s. :) Much more readable imo, perhaps not here, but when you get into more complicated structures it makes all the difference.

Reply:
Done